tests/Fixtures/index.md

```diff
@@ -39,20 +39,20 @@ curl -X GET -G "http://localhost/api/test" \
 ```
 
 ```javascript
-const headers = new Headers({'Accept': 'application/json'})
+const headers = new Headers({ Accept: "application/json" });
 
 const settings = {
-    method: "GET"
-    credentials: 'include'
-    headers,
-}
+    method: "GET",
+    credentials: "include",
+    headers
```

Comment:
Why did you remove the comma? The build's failing.

Reply:
i've forgot to write comma